compiler/rustc_target/src/spec/mod.rs

```diff
@@ -954,6 +956,7 @@ impl ToJson for TlsModel {
             TlsModel::LocalDynamic => "local-dynamic",
             TlsModel::InitialExec => "initial-exec",
             TlsModel::LocalExec => "local-exec",
+            TlsModel::EmulatedTls => "emulated-tls",
```

Comment:
```suggestion
            TlsModel::Emulated => "emulated",
```
Is the tautology necessary?
`-Z tls-model=emulated` seems fine as is.